pom.xml

```diff
@@ -456,7 +457,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <argLine>@{argLine} -Djava.awt.headless=true</argLine>
+                    <argLine>@{argLine} -Djava.awt.headless=true ${maven-surefire-plugin.vmopts}</argLine>
```

Comment:
How about just setting it to 128mb as the default for everything? It's not much difference and avoids the need to set this individually for different repos that might come up as a problem later.

Reply:
fixed above. thanks!